src/app/app.component.html

```diff
@@ -1,9 +1,25 @@
-<router-outlet></router-outlet>
+<mat-toolbar color="primary">
+  <h1>Welcome to Taskilicious</h1>
+  <span class="topbar-flex"></span>
+  <button
```

Comment:
Hmm, I don't like it. You are moving some logic of categories to app.component, why can't you just add this buutton to category component?

Reply:
This is only a toolbar with basic navigation, no real logic is present here; this would normally be a separate component, but the project is small so this is OK I think

Reply:
At the end I will be taking a look at the project as a whole and will probably change this part